### PR TITLE
Joss

### PR DESCRIPTION
**Pull request by mhesselbarth**

### CODE REVIEW

File: R/results_habitat_association.R [hunk 1]
```diff
@@ -150,6 +150,13 @@ results_habitat_association <- function(pattern, raster,
 
     }
 
+    # warning if NA are present
+    if (anyNA(raster@data@values)) {
```
Comment (Bisaloo) [inline R/results_habitat_association.R:154]: This adds a dependency to R >= 3.2.0

Comment (mhesselbarth) [reply R/results_habitat_association.R:154]: Oh really? I didn't know that. Fixed in Description (3f9c5bb).

**Comment by mhesselbarth** [comment]
Not merging yet. Just preparing and making sure all GitHub actions run and no conflicts.

File: R/results_habitat_association.R (hunk 1 above)
Comment (Bisaloo) [reply R/results_habitat_association.R:154]: Oh my bad, you're right. After looking into the release notes, `anyNA()` was added in R 3.1.0. R 3.2.0 says:

> anyNA() gains a recursive argument.

The release notes are here: https://cran.r-project.org/doc/manuals/NEWS.3

Comment (mhesselbarth) [reply R/results_habitat_association.R:154]: It requires R>=3.1.0 though

Comment (Bisaloo) [reply R/results_habitat_association.R:154]: Yes, it does

**Pull request merged**